lib/govuk_request_id.rb

```diff
@@ -0,0 +1,13 @@
+class GovukRequestId
+  def self.insert(body)
+    return body unless body.present?
+
+    doc = Nokogiri::HTML::DocumentFragment.parse(body)
```

Comment:
We'd have to check that the comments got through the rest of the stack. There's also the issue of plain text emails. I don't know if we send them and if we do I don't know whether it would necessarily be a problem to append an identifier.